synthtool/gcp/gapic_bazel.py

```diff
@@ -95,6 +96,15 @@ def _generate_code(
                 f"Unable to generate {service}, the sources repository repository"
                 "is unavailable."
             )
+        if not bazel_target or include_protos:
```

Comment:
Add a comment describing what the next 8 lines of code are doing.

Reply:
I refactored that section, hopefully making it easier to understand, and added the comments. PTAL.

Reply:
Looks great.  Thanks for the changes.